composer.json

```diff
@@ -9,14 +9,31 @@
             "EventSourcingWorkshop\\": "src"
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+            "EventSourcingWorkshopTest\\": "test"
+        }
+    },
     "require": {
         "php": "^8.1.8",
```

Comment:
a `Dockerfile` should be provided.

 * [ ] docs needed for how to use that
 * [ ] in case network fails, a pre-built docker container should be provided to attendees (USB stick)

Reply:
`Dockerfile` provided - need to push up a stable image, and add docs for it.